packages/twenty-server/src/database/commands/upgrade-version/0-24/0-24-migrate-email-fields-to-emails.command.ts

```diff
@@ -0,0 +1,325 @@
+import { Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+
+import chalk from 'chalk';
+import { Command, CommandRunner, Option } from 'nest-commander';
+import { QueryRunner, Repository } from 'typeorm';
+
+import { TypeORMService } from 'src/database/typeorm/typeorm.service';
+import { DataSourceEntity } from 'src/engine/metadata-modules/data-source/data-source.entity';
+import { DataSourceService } from 'src/engine/metadata-modules/data-source/data-source.service';
+import { CreateFieldInput } from 'src/engine/metadata-modules/field-metadata/dtos/create-field.input';
+import {
+  FieldMetadataEntity,
+  FieldMetadataType,
+} from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { FieldMetadataService } from 'src/engine/metadata-modules/field-metadata/field-metadata.service';
+import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
+import { TwentyORMGlobalManager } from 'src/engine/twenty-orm/twenty-orm-global.manager';
+import { computeTableName } from 'src/engine/utils/compute-table-name.util';
+import { WorkspaceStatusService } from 'src/engine/workspace-manager/workspace-status/services/workspace-status.service';
+import { ViewService } from 'src/modules/view/services/view.service';
+import { ViewFieldWorkspaceEntity } from 'src/modules/view/standard-objects/view-field.workspace-entity';
+
+interface MigrateEmailFieldsToEmailsCommandOptions {
+  workspaceId?: string;
+}
+
+@Command({
+  name: 'upgrade-0.24:migrate-email-fields-to-emails',
+  description: 'Migrating fields of deprecated type EMAIL to type EMAILS',
+})
+export class MigrateEmailFieldsToEmailsCommand extends CommandRunner {
```

Comment:
FYI you can extend `ActiveWorkspacesCommandRunner` and remove some boilerplate.
You can check an example in `SetCustomObjectIsSoftDeletableCommand`

Reply:
ok, thanks, we are looking into it